.env.sample

```diff
@@ -1,4 +1,4 @@
-VITE_API_URL=<API_URL>
+# VITE_API_URL=https://api-dev.sunflower-land.com
```

Comment:
We don't want to expose this URL in our repo - it makes it an easy target for bad actors 😓

Reply:
Done